scalding-serialization-macros/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/TreeOrderedBuf.scala

```diff
@@ -122,15 +128,21 @@ object TreeOrderedBuf {
         case const: ConstantLengthCalculation[_] => (q"""
           override val staticSize: Option[Int] = Some(${const.toInt})""", q"""
           override def dynamicSize($element: $typeName): Option[Int] = staticSize""")
-        case f: FastLengthCalculation[_] => callDynamic
-        case m: MaybeLengthCalculation[_] => callDynamic
+        case f: FastLengthCalculation[_] =>
+          callDynamic
+        case m: MaybeLengthCalculation[_] =>
+          callDynamic
+        case _ => (q"""
```

Comment:
I should/will revert this. I was moving towards only allowing callDynamic being used here if the length of the code in it was less than some threshold. In practice that proved too hard to tune though.